ui/app/src/services/configuration.ts

```diff
@@ -198,7 +199,7 @@ export function getAudiencesPanelModel(): Observable<ContentInstance> {
   }));
 }
 
-export function setAudiencesPanelModel(params): Observable<any> {
+export function setActiveProfile(params): Observable<any> {
```

Comment:
1. Type params and return type
2. Return the content of the AjaxResponse

`.get(‘...’).pipe(pluck(‘response’))`